sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -1490,8 +1536,8 @@ def __init__(
       schema (str,dict,ValueProvider,callable): The schema to be used if the
         BigQuery table to write has to be created. This can be either specified
         as a :class:`~apache_beam.io.gcp.internal.clients.bigquery.\
-bigquery_v2_messages.TableSchema`. or a `ValueProvider` that has a JSON string,
-        or a python dictionary, or the string or dictionary itself,
+        bigquery_v2_messages.TableSchema`. or a `ValueProvider` that has a JSON
```

Comment:
I think this may be causing the docs test failure

Reply:
From the test suite:
```
00:02:30.217 /home/jenkins/jenkins-slave/workspace/beam_PreCommit_PythonDocs_Commit/src/sdks/python/test-suites/tox/pycommon/build/srcs/sdks/python/apache_beam/io/gcp/bigquery.py:docstring of apache_beam.io.gcp.bigquery.WriteToBigQuery:28: WARNING: py:class reference target not found: apache_beam.io.gcp.internal.clients.bigquery.        bigquery_v2_messages.TableSchema
```

Reply:
Thanks for pointing it out! Fixed and pushed a new commit.